tools/docs/github-query-items.ts

```diff
@@ -46,6 +29,34 @@ export interface Items {
   features: ItemsEntity[];
 }
 
+const GhOutputSchema = z.array(
+  z.object({
+    url: z.string(),
+    title: z.string(),
+    labels: z.array(
+      z.object({
+        name: z.string(),
+      }),
+    ),
+    number: z.number(),
+  }),
+);
+
+async function getIssuesByIssueType(
+  issueType: 'Bug' | 'Feature',
+): Promise<ItemsEntity[]> {
+  const command = `gh issue list --json "title,number,url,labels" --search "type:${issueType}" --limit 1000`;
```

Comment:
why the limit to 1000? we didn't had it before 🤔

can we change page sizes to 100?

Reply:
I made the limit 1000 cause we have 600+ issues currently, so we can get the json all the issues in one command execution instead of making 7 executions.